modules/ROOT/pages/managing-instances/instance-actions.adoc

```diff
@@ -200,3 +200,21 @@ image::mark-as-production.png[]
 [.shadow]
 .Instance marked as production
 image::marked-as-production.png[]
+
+== Upgrade AuraDB Professional to AuraDB Business Critical
+
+To upgrade your AuraDB Professional instance to AuraDB Business Critical, select the more menu (three dots) on an instance card, then select *Upgrade*.
+
+.Upgrade your AuraDB Professional instance to AuraDB Business Critical
+[.shadow]
+image::upgradeprotobc.png[]
+
+This upgrade does not change the original DBID, so your application strings remain valid.
+It enables the capabilities that would otherwise be available with an AuraDB Business Critical instance. 
```

Comment:
This is confusing. What enables these capabilities? The upgrade? The retained DBID?
Also, _capabilities that would otherwise be available with and AuraDB Business Critical instance_ is also problematic. I mean, if you upgrade from Pro to BC, you would expect to get the capabilities of BC, right?